content/pt-br/docs/reference/glossary/pod-disruption.md

```diff
@@ -0,0 +1,21 @@
+---
+id: pod-disruption
+title: Interrupção do Pod
+full_link: /docs/concepts/workloads/pods/disruptions/
+date: 2021-05-12
+short_description: >
+  O processo pelo qual Pods ou Nós são interrompidos de forma voluntária ou involuntária.
```

Comment:
Minor: it would be better to write "nós" with a small "n" here.

Reply:
Done!